### PR TITLE
AT-SCE-UI 24 Admin Form Validations / Error Handling

### PR DESCRIPTION
**Pull request by Emiliano-Plascencia**
We added support to handle error validations from BE.
We deleted the catch error in the operations create, update and delete user from the service file.
In the admin.controller.js file we have added redirects when a user is created, updated and deleted.
Added missing "password" label to update a user and the changes have been implemented in the admin.controller.js file as well

### CODE REVIEW

File: src/views/admin/user/edit-user.hbs [hunk 1]
```diff
@@ -51,6 +51,15 @@
                         <input type="text" name="user_email" id="user_email" class="form-control" value="{{user.email}}"
                             placeholder="Type an Email" required />
                     </div>
+                    <div class="form-group">
+                        <label for="user_password"> Password:</label>
+                        <input type="password" name="user_password" id="user_password" class="form-control"
+                            placeholder="Type a Password" min="8" max="20" />
```
Comment (at-internship) [inline src/views/admin/user/edit-user.hbs]: `placeholder="Type a Password" min="10" max="20" pattern="^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=\\S+$).{10,}$" />`

File: src/views/admin/user/edit-user.hbs [hunk 2]
```diff
@@ -51,6 +51,15 @@
...
+                        <!--small id="passwordHelpBlock" class="form-text text-muted">
```
Comment (at-internship) [inline src/views/admin/user/edit-user.hbs]:
```
<small id="passwordHelpBlock" class="form-text text-muted">
  Your password must be 10-20 characters long, contain at least one uppercase character, contain at least one lowercase character, at least one numerical character, and must not contain spaces, special characters.
<small>
```

(Also add this on add-user.hbs)

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/views/admin/user/edit-user.hbs [hunk 3]
```diff
@@ -51,6 +51,15 @@
                         <input type="text" name="user_email" id="user_email" class="form-control" value="{{user.email}}"
                             placeholder="Type an Email" required />
```
Comment (at-internship) [inline src/views/admin/user/edit-user.hbs]:
```
<input type="text" name="user_email" id="user_email" class="form-control" value="{{user.email}}" 
placeholder="Type an Email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" /> 
```

(Also add this on add-user.hbs)

Comment (luishcastroc) [reply src/views/admin/user/edit-user.hbs]: isn't it easy with type=email?

File: src/views/admin/user/edit-user.hbs (hunk 1 above)
Comment (Emiliano-Plascencia) [reply src/views/admin/user/edit-user.hbs:57]: I did that change but now I can't save the password, it shows a message "Utiliza un formato que coincida con el solicitado" and I am typing the password with the correct conditions.

File: src/views/admin/user/edit-user.hbs [hunk 4]
```diff
@@ -48,8 +48,17 @@
                     </div>
                     <div class="form-group">
                         <label for="user_email">* Email:</label>
-                        <input type="text" name="user_email" id="user_email" class="form-control" value="{{user.email}}"
-                            placeholder="Type an Email" required />
+                        <input type="text" name="user_email" id="user_email" class="form-control" value="{{user.email}}"
```
Comment (at-internship) [inline src/views/admin/user/edit-user.hbs]: `<input type="email"`

File: src/views/admin/user/add-user.hbs [hunk 5]
```diff
@@ -46,17 +46,17 @@
                     </div>
                     <div class="form-group">
                         <label for="user_email">* Email:</label>
-                        <input type="text" name="user_email" id="user_email" class="form-control"
-                            placeholder="Type an Email" required />
+                        <input type="text" name="user_email" id="user_email" class="form-control" value="{{user.email}}"
```
Comment (at-internship) [inline src/views/admin/user/add-user.hbs]: <input type="email"

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by at-internship — APPROVED** [review]

**Review by FlavioAlejandro — APPROVED** [review]

**Review by DianaArceoSuarez — APPROVED** [review]

**Pull request merged**